### ISSUE TITLE
OpenPose does not show anything on my desktop

### ISSUE THREAD

**Issue by flowerdanceee** [CMU-Perceptual-Computing-Lab/openpose#1940]
I think that I have successfully compile openpose on my desktop.But error occurs.
![Screenshot from 2021-05-11 01-10-40](https://user-images.githubusercontent.com/55842731/117690410-c9762880-b1f5-11eb-9e1c-a5df82c4fd8a.png)
Is there someone can help me?thank you very much.
here is the imformation of my desktop
os:Ubuntu 20.04
GPU:rtx 3070
drivers version:460.73
![Screenshot from 2021-05-11 01-16-14](https://user-images.githubusercontent.com/55842731/117691244-a5671700-b1f6-11eb-9fd9-4a0ecc61bb3d.png)
cuda:11.1.1
![Screenshot from 2021-05-11 01-13-49](https://user-images.githubusercontent.com/55842731/117691316-b9127d80-b1f6-11eb-8512-0c4c724f896f.png)
cudnn:8.0.4
![Screenshot from 2021-05-11 01-15-16](https://user-images.githubusercontent.com/55842731/117691351-c3cd1280-b1f6-11eb-8f5f-8bd329ba615f.png)

**Comment by ravijo** [comment]
@flowerdanceee 

It seems you are facing the following error:

```
cudnn_conv_layer.cpp:53] Check failed: status == CUDNN_STATUS_SUCCESS (1 vs 0) CUDNN_STATUS_NOT_INITIALIZED
```

The problem could be related to CUDA version compatibility as said [here](https://github.com/CMU-Perceptual-Computing-Lab/openpose/issues/1508#issuecomment-592980826)

Therefore, can you please read [this document](https://github.com/CMU-Perceptual-Computing-Lab/openpose/blob/master/doc/installation/1_prerequisites.md) to confirm version compatibility before proceeding further?


Seeing [this post](https://github.com/CMU-Perceptual-Computing-Lab/openpose/issues/1527#issuecomment-609058873), the error could be related to limited GPU memory based on your OpenPose configuration.

Finally, I suggest you to decrease net_resolution and check once more.

**Comment by flowerdanceee** [comment]
> @flowerdanceee
> 
> It seems you are facing the following error:
> 
> ```
> cudnn_conv_layer.cpp:53] Check failed: status == CUDNN_STATUS_SUCCESS (1 vs 0) CUDNN_STATUS_NOT_INITIALIZED
> ```
> 
> The problem could be related to CUDA version compatibility as said [here](https://github.com/CMU-Perceptual-Computing-Lab/openpose/issues/1508#issuecomment-592980826)
> 
> Therefore, can you please read [this document](https://github.com/CMU-Perceptual-Computing-Lab/openpose/blob/master/doc/installation/1_prerequisites.md) to confirm version compatibility before proceeding further?
> 
> Seeing [this post](https://github.com/CMU-Perceptual-Computing-Lab/openpose/issues/1527#issuecomment-609058873), the error could be related to limited GPU memory based on your OpenPose configuration.
> 
> Finally, I suggest you to decrease net_resolution and check once more.

Thank you for your reply！

I‘m sure that there is only one cuda/cudnn in my desktop, because every time I change the cuda/cudnn version, I will reinstall the Ubuntu system to ensure that there is only one cuda/cudnn in my desktop.

In addition, I did find the problem of insufficient graphics card memory. But what I don't understand is that rtx3070 has 8G but it is still not enough. Now we are considering whether we need to add another graphics card.

**Comment by ravijo** [comment]
@flowerdanceee 

> In addition, I did find the problem of insufficient graphics card memory. But what I don't understand is that rtx3070 has 8G but it is still not enough. Now we are considering whether we need to add another graphics card.

Probably you are running OpenPose with face and hand detection too. In general 8GB is enough for default OpenPose to run. Why don't you execute OpenPose with the following configuration
- Disable face (and hand too) 
- Lower the net resolution

You can find more information on lowering the memory usage on the OpenPose documentation page.

**Comment by flowerdanceee** [comment]
> @flowerdanceee
> 
> > In addition, I did find the problem of insufficient graphics card memory. But what I don't understand is that rtx3070 has 8G but it is still not enough. Now we are considering whether we need to add another graphics card.
> 
> Probably you are running OpenPose with face and hand detection too. In general 8GB is enough for default OpenPose to run. Why don't you execute OpenPose with the following configuration
> 
> * Disable face (and hand too)
> * Lower the net resolution
> 
> You can find more information on lowering the memory usage on the OpenPose documentation page.

Thank you for your reply!

1.Disable face (and hand too)
I use the command line which is [./build/examples/openpose/openpose.bin --video examples/media/video.avi] to run openpose.It does not seem to get the keypoints the hands and face.

2.Lower the net resolution
Refering to [this page](https://github.com/CMU-Perceptual-Computing-Lab/openpose/blob/master/doc/01_demo.md#improving-memory-and-speed-but-decreasing-accuracy) ,I have modified the size of the parameter, but it still can’t run.

**Comment by ravijo** [comment]
@flowerdanceee

Hmmmm

I highly doubt if it is related to CUDA (or Caffe compatibility). I am sorry that there is no easy way to find it out. The `CUDNN_STATUS_NOT_INITIALIZED` error given by Caffe is a generic, which I have faced previously because of compatibility issue.

**Comment by flowerdanceee** [comment]
> @flowerdanceee
> 
> Hmmmm
> 
> I highly doubt if it is related to CUDA (or Caffe compatibility). I am sorry that there is no easy way to find it out. The `CUDNN_STATUS_NOT_INITIALIZED` error given by Caffe is a generic, which I have faced previously because of compatibility issue.

Thank you for your reply!

Is it possible that it is just the problem of insufficient graphics card memory? Or the memory consumed on the rtx30 series of graphics cards is relatively large? I have used [nvidia-smi -l 1] to check the graphics card memory, but it was full in an instant indeed.

**Comment by ravijo** [comment]
@flowerdanceee 

> Is it possible that it is just the problem of insufficient graphics card memory? 

Yes. This problem is faced by some uses in case of insufficient graphics card memory.

> Or the memory consumed on the rtx30 series of graphics cards is relatively large? 

I am not sure. I have not used the RTX30 series. But it seems unlikely that the RTX30 series consumes more memory compared to others. I am not sure.

> I have used [nvidia-smi -l 1] to check the graphics card memory, but it was full in an instant indeed.

I see. Probably, other processes have already eaten a large amount of graphics card memory, and then RTX is left with a little unallocated memory. In general, for OpenPose, you should have at least 4GB of free memory in the graphics card.

**Comment by Kitoali** [comment]
> 
some how what shloud we do to fix it.

**Comment by ravijo** [comment]
@Kitoali 

As said earlier, please check your Caffe installation. The problem is apparently related to the Caffe library and there are many ways to install it.
